WorkAndFun/src/main/java/pl/infoshare/workandfun/announcements/AnnouncementService.java

```diff
@@ -72,16 +76,68 @@ public Announcement update(Long id, AddAndEditAnnouncementDto dto) {
+                ServiceType.BUDOWA_DOMU_BUDOWA_OD_PODSTAW, "Wrocław", "", "", "50000", null, Voivodeship.DOLNOSLASKIE, null, "Piotrek",
                 "piotrek@b.pl", true, "Nie masz chaty? Zbuduję! Mach chatę? Zbuduję Ci nową! Nie krępuj się i dzwoń śmiało!"
                 , "+48777777777", ""));
         announcementsRepository.save(new Announcement(3L, Type.SERVICE_OFFER, "Profesjonalne obieranie cebuli",
-                ServiceType.INNE, "Gdańsk","","", "10", null, Voivodeship.POMORSKIE, null,"Polcebulex",
+                ServiceType.INNE, "Gdańsk", "", "", "10", null, Voivodeship.POMORSKIE, null, "Polcebulex",
                 "polcebulex@wp.pl", false, "1kg obranej cebuli za 10gr. Czas realizacji na zamówienia do pół tony to 5 dni roboczych." +
                 "Zapraszamy do składania zamówień w wiodącej firmie obierającej cebulę w regionie pomorskim.", "+48888888888", ""));
+
+        announcementsRepository.save(new Announcement(
+                4L,
+                Type.SERVICE_OFFER,
+                "Wkręcam żarówki TANIO!",
+                ServiceType.ELEKTRYK,
+                "Wrocław",
+                "Krzyki",
+                "Powstańców Śląskich",
+                "10",
+                null,
+                Voivodeship.DOLNOSLASKIE,
+                null,
+                "PiotrPiotrowski",
+                "example@gmail.com",
+                false,
+                "Wkręcam żarówki.",
+                "+48123456789",
+                "cena za sztukę"));
+
+        announcementsRepository.save(new Announcement(
+                5L,
+                Type.SERVICE_OFFER,
+                "Diagnostyka i serwis aut elektrycznych",
+                ServiceType.MOTORYZACJA,
+                "Legnica",
+                "",
+                "",
+                "do ustalenia indywidualnie",
+                null,
+                Voivodeship.DOLNOSLASKIE,
+                null,
+                "Leszek",
+                "rexoAuto@gmail.com",
+                false,
+                "Diagnostyka i serwis aut elektrycznych. Pomagam przy zakupie auta elektrycznego",
+                "+48741741741",
+                ""));
+    }
+
+    public Iterable<QuickViewAnnouncementDto> searchAllByParameter(String parameter) {
+        List<QuickViewAnnouncementDto> announcementDtoList = (List<QuickViewAnnouncementDto>) findAllSortedByCreateDateDescConvertToDto();
+        if (parameter.isBlank()) {
+            return announcementDtoList;
+        } else {
+            return announcementDtoList.stream()
+                    .filter(searchFilter(parameter)).collect(Collectors.<QuickViewAnnouncementDto>toList());
+        }
+    }
+
+    private Predicate<QuickViewAnnouncementDto> searchFilter(String param) {
+        return quickViewAnnouncementDto -> quickViewAnnouncementDto.toString().toLowerCase().contains(param.toLowerCase());
     }
```

Comment:
proszę przenieść te metody ponad metodę, która dodaje ogłoszenia do DB, będzie to bardziej widoczne i czytelne. Reszta wyglada ok